core-feature-pack/pom.xml

```diff
@@ -120,6 +120,11 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.spec.javax.security.auth.message</groupId>
+            <artifactId>jboss-jaspi-api_1.1_spec</artifactId>
+        </dependency>
+
```

Comment:
I understand this dependency should be added to core-galleon-pack/pom.xml as well. Galleon won't be able to include it in the module if it's not in that pom.xml file.

Reply:
Added, it was in a later commit on the topic branch but does need to be included in this PR.